### PR TITLE
Add KYC settings

### PR DESCRIPTION
**Pull request by subenson**

### CODE REVIEW

File: openapi/code_samples/JavaScript/settings@kyc/get.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const kycSettings = await api.kycSettings.get();
+console.log(kycSettings.requireDateOfBirth);
```
Comment (p3yman) [inline openapi/code_samples/JavaScript/settings@kyc/get.js:2]: Is this for debugging or should be removed?

File: openapi/components/schemas/Common/CommonKycDocument.yaml [hunk 2]
```diff
@@ -56,6 +56,9 @@ properties:
     description: Processing date/time.
     allOf:
       - $ref: ../ServerTimestamp.yaml
+  settings:
+    readOnly: true
+    $ref: ../KycSettings.yaml
```
Comment (slavcodev-rebilly) [inline openapi/components/schemas/Common/CommonKycDocument.yaml:61]: This is not correct schema according OAS 3.0.3 we use. if `$ref` keyword is used, it it should single in that level. You can use this instead if composition is needed
```yaml
readOnly: true
allOf:
    - $ref: ../KycSettings.yaml
```

Comment (slavcodev-rebilly) [reply openapi/components/schemas/Common/CommonKycDocument.yaml:61]: BTW, if this settings are read-only, how we can set them?

Comment (subenson) [reply openapi/components/schemas/Common/CommonKycDocument.yaml]: For now it's set manually for merchants. We hammered the scope of our project.

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**